app/models/assign_mailer_site_settings.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require_dependency 'enum_site_setting'
+
+class AssignMailerSiteSettings < EnumSiteSetting
+
+  def self.valid_value?(val)
+    values.any? { |v| v[:value].to_s == val.to_s }
+  end
+
+  def self.values
+    @values ||= [
+      { name: 'discourse_assign.assign_mailer.never', value: 'never' },
+      { name: 'discourse_assign.assign_mailer.different', value: 'different' },
```

Comment:
what does "different" mean?

Reply:
It means that the assign mailer will be send an email only if the assigner and assignee are different. Regis recommended me to change this to `different_users`, so I did it. I hope everything is clear now.